frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -33,7 +33,9 @@
         vm.stopLeaderboard = function() {};
         vm.stopFetchingSubmissions = function() {};
         vm.currentDate = null;
-
+        vm.sortColumn = 'score';
```

Comment:
@vladan-jovicic can we make this dynamic? The label name comes from the backend so there can be multiple labels. For example, VQA Challenge has 4 labels `yes/no`, `number`, `other`, `overall`. 

To get the labels for a particular challenge, you will have to fetch the label name from Leaderboard API for the particular challenge.

Reply:
I think that your approach is a lot neater and we can go with this. One minor request, can we change `columnSort = 'score'` to `columnSort = 'number'` and `sortColumn === 'team'` to `sortColumn === 'string'` for better readability.